src/CSF.SRDashboard.Client/Shared/NavMenu.razor.css

```
@@ -18,37 +18,44 @@
     top: -2px;
 }
 
+.fa-search {
+    font-size: 18px;
+}
+
+.fa-clipboard {
+    font-size: 18px;
+}
+.fa-fw {
+    text-align: left;
+    width: 1.25em;
+}
+
 .nav-item {
-    font-size: 0.9rem;
-    padding-bottom: 0.5rem;
+    padding-left: 7px;
+    font-size: 16px;
+    letter-spacing: 0.01em;
```

For padding we can add bootstrap classes to the .nav-item element. https://getbootstrap.com/docs/5.0/utilities/spacing/

We also don't want to add font sizes in pixels. We want them to be in terms of rem. In this case we replace 16px with 1rem;

In this case we also want the letter-spacing to be in rem terms.
```suggestion
    font-size: 1rem;
    letter-spacing: 0.01rem;
```